libs/community/langchain_community/document_loaders/unstructured.py

```diff
@@ -219,6 +224,53 @@ def __init__(
 
         super().__init__(mode=mode, **unstructured_kwargs)
 
+    def lazy_load(self) -> Iterator[Document]:
+        """Load file."""
+        # This method overwrites the UnstructuredBaseLoader method because that one expects
+        # `Element` objects instead of a json, which is what the SDK returns.
+        elements_json = self._get_elements()
+        self._post_process_elements(elements_json)
+
+        if self.mode == "elements":
+            for element in elements_json:
+                metadata = self._get_metadata()
+                # # NOTE(MthwRobinson) - the attribute check is for backward compatibility
+                # # with unstructured<0.4.9. The metadata attributed was added in 0.4.9.
+                if element.get("metadata") is not None:
+                    metadata.update(element["metadata"])
+                # if hasattr(element, "category"):
+                #     metadata["category"] = element.category
+                yield Document(page_content=element.get("text"), metadata=metadata)
+        elif self.mode == "paged":
```

Comment:
@MthwRobinson, this `"paged"` mode seems to be equivalent to the `by_page` chunking. I thought about updating the request in the sdk if `self.mode == "paged"`, but I am wondering if there would be any issue with changing this process from post-processing like it is now. Do you think that's safe to do?

Reply:
I would leave this to avoid breaking things for people who are using it. Of course, `chunking_strategy="by_page"` is fine to pass in as well.